addons/api/tests/unit/models/target-test.js

```diff
@@ -384,4 +384,24 @@ module('Unit | Model | target', function (hooks) {
     const model = store.peekRecord('target', '123abc');
     await model.removeCredentialSource('2');
   });
+
+  test('it has isSSH property and returns the expected values', async function (assert) {
+    assert.expect(2);
+    const store = this.owner.lookup('service:store');
+    const modelSSH = store.createRecord('target', {
+      type: 'ssh',
+    });
+    assert.true(modelSSH.isSSH);
+    assert.false(modelSSH.isTCP);
+  });
+
+  test('it has isTCP property and returns the expected values', async function (assert) {
```

Comment:
```suggestion
  test('it has isTCP property and returns the expected values', function (assert) {
```
Same as other comment.